src/components/Notification/Notification.js

```diff
@@ -42,20 +42,17 @@ export class NotificationButton extends Component {
       notificationType,
       ...other
     } = this.props;
-
     const buttonClasses = classNames(
       {
         'bx--toast-notification__close-button': notificationType === 'toast',
         'bx--inline-notification__close-button': notificationType === 'inline',
       },
       className
     );
-
     const iconClasses = classNames({
-      'bx--toast-notification__icon': notificationType === 'toast',
+      'bx--toast-notification-icon': notificationType === 'toast',
```

Comment:
Looks that this change drifts away from https://github.com/IBM/carbon-components/blob/v9.46.0/src/components/notification/_toast-notification.scss#L139?

Reply:
I had some questions about that:

if I'm not mistaken, it looks like the component is using `bx--toast-notification-icon` here https://github.com/IBM/carbon-components/blob/v9.46.0/src/components/notification/notification.hbs#L88

do we want to sync the classes and styles on this component for the two libraries?

vanilla notification
![image](https://user-images.githubusercontent.com/8265238/48206915-1af78100-e335-11e8-9f41-6bb927bdffd6.png)

React `bx--toast-notification-icon`
![image](https://user-images.githubusercontent.com/8265238/48206948-32366e80-e335-11e8-974d-fe74881007ce.png)

React `bx--toast-notification__icon`
![image](https://user-images.githubusercontent.com/8265238/48206983-467a6b80-e335-11e8-8e76-2784902a82b5.png)

based off of the `bx--inline-notification__close-icon` class, is it supposed to be `bx--toast-notification__close-icon` in both repos?

Reply:
going to go with `bx--toast-notification__close-icon` to follow @tw15egan's latest changes here https://github.com/IBM/carbon-components/pull/1412